src/utils/faqArray.ts

```diff
@@ -1,20 +1,61 @@
 export const faqArray = [
   {
     id: 0,
-    question: "question 1",
-    answer:
-      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat eligendi enim magni nesciunt atque nam nemo natus unde ea quis, quaeitaque dolores, rem totam accusantium beatae explicabo quos quod!",
+    question: "Payment at the Machine:",
```

Comment:
Take out the semi-colons of all the questions
![image](https://github.com/Kenzie-Tech-Solutions/theRoyalCanine/assets/88916004/80bca7cb-f937-49b9-8390-5ba3832be0dd)